src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  const [searchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex');
+  const query = searchParams.get('query');
+  const centuries = searchParams.getAll('centuries');
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const filteredPeople = useMemo(() => {
+    let result = [...people];
+
+    if (sex) {
+      result = result.filter(person => person.sex === sex);
+    }
+
+    if (query) {
+      const queryNormalize = query.trim().toLowerCase();
+
+      result = result.filter((person) => {
+        return (
+          person.name.toLowerCase().includes(queryNormalize)
+        || person.motherName?.toLowerCase().includes(queryNormalize)
+        || person.fatherName?.toLowerCase().includes(queryNormalize));
+      });
+    }
+
+    if (centuries.length) {
+      result = result.filter((person) => {
+        return (
+          centuries.includes(Math.ceil(person.born / 100).toString())
+        );
+      });
+    }
+
+    if (sort) {
+      result = result.sort((a, b) => {
+        switch (sort) {
+          case 'Name':
+            return a.name.localeCompare(b.name);
+          case 'Sex':
+            return a.sex.localeCompare(b.sex);
+          case 'Born':
+            return a.born - b.born;
+          case 'Died':
+            return a.died - b.died;
```

Comment:
The same as above, move these words to enum and use it here
```suggestion
          case 'Name':
            return a.name.localeCompare(b.name);
            
          case 'Sex':
            return a.sex.localeCompare(b.sex);
            
          case 'Born':
            return a.born - b.born;
            
          case 'Died':
            return a.died - b.died;
            
```